feature/main/ui/src/main/kotlin/com/epiclabs/thinktok/main/ui/languageselection/SubmitAndContinueButton.kt

```diff
@@ -0,0 +1,24 @@
+package com.epiclabs.thinktok.main.ui.languageselection
+
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+internal fun SubmitAndContinueButton(
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit = {},
+    buttonText: String,
```

Comment:
Modifier should be first optional argument after required ones.

```suggestion
    buttonText: String,
    modifier: Modifier = Modifier,
    onClick: () -> Unit = {},
```